src/snowflake/snowpark/session.py

```diff
@@ -2749,4 +2749,32 @@ def _explain_query(self, query: str) -> Optional[str]:
             _logger.warning("query `%s` cannot be explained", query)
             return None
 
+    @staticmethod
+    def create() -> "Session":
+        """Gets the existing session if one exists (in case of SPs/Streamlits) or creates a new one using default connection params (e.g., in case of SPCS)."""
+
+        # 1) check if in SiS or sproc
+        if (
+            is_in_stored_procedure()
+        ):  # (will be removed before merging) no need for `"streamlit" in sys.modules` check since is_in_stored_procedure() must be true for SiS as well
+            # Basically noop and return the existing session
+            return _get_active_session()
+        try:
+            # 2) check if we are running in SPCS
+            with open("/snowflake/session/token") as file:
```

Comment:
Should we create a `is_in_spcs` util function? This way we don't need to include `except` in the normal code flow.

Reply:
Done